add: cargo install documentation (#14)
DarthBenro008/rchore: README.md

### Cargo

If you have cargo installed, you can install rChore using cargo.

> - A number of settings should be set via environment variables. To obtain `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` follow to: https://developers.google.com/tasks/firstapp#register. To set a variables you should add the following snippets (using your values for vars) to your ~/.bashrc / ~/.zshrc / ~/.config/fish/config.fish

Reviewer: ```suggestion
- Few secret keys should be set via environment variables. To obtain `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` follow to: https://developers.google.com/tasks/firstapp#register. To set these variables you should add the following snippets (using your values for vars) to your `~/.bashrc` / `~/.zshrc` / `~/.config/fish/config.fish`
```

> bash / zsh:

Reviewer: It would be nice if its in bold/points

```bash
export GOOGLE_CLIENT_ID=xxx
export GOOGLE_CLIENT_SECRET=yyy
```

> fish:

Reviewer: It would be nice if its in bold/points
